packages/esm-patient-label-printing-app/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@openmrs/esm-patient-label-printing-app",
```

Comment:
If we plan to use this app to print a wider range of documents, then `@openmrs/esm-patient-printing-app` would be a more generic name.